RocketChat/Rocket.Chat
# [FIX] Click to join button Jitsi Call

## Proposed changes (including videos or screenshots)


Added `ToolboxProvider` to `MessageListProvider` and fixed actionLink.js open function exec
 


## Issue(s)

Click to join in jitsi call is not working

## Steps to test or reproduce

Steps to reproduce:

1. Log in to the [rocket.chat](http://rocket.chat/) as user1
2. Open a room (DM/Discussion/Channel/Team)
3. Click on the icon phone to start a new Video Conference 
4. Log in to the [rocket.chat](http://rocket.chat/) as user2
5. Open the same room and click on click to Join 

Actual result:
Nothing happens

Expected result:
  Should be opened the video call

## Further comments
